test/wot-nosecurityscheme/mapping.yarrrml

```diff
@@ -0,0 +1,23 @@
+sources:
+  people-source:
+    access: "http://example.org/file.xml"
+    contentType: "application/xml"
+    operationType: read
+    type: wot
+    referenceFormulation: xpath
+    iterator: "/"
+    security:
+      - type: none
```

Comment:
Can't we just remove lines 9 and 10 then?

Reply:
We could... But the WoT spec allows to specify `NoSecurityScheme` explicitly. That's why I wanted to include it.
There's another one with `without-security` which drops lines 9-10, but doesn't generate any security description then.

Reply:
Ok, make sense then, but maybe we can do in the above case have a shortcut like `security: none`?

Reply:
I can introduce a shortcut, but I would keep the expanded form of course :)

Reply:
Of course, yeah, keep the expanded form.

Reply:
@pheyvaer Added shortcut and a test for it.